src/pip/_internal/wheel_builder.py

```diff
@@ -375,7 +384,11 @@ def _build_one_inside_env(
                                 wheel_hash.hexdigest())
                     logger.info('Stored in directory: %s', output_dir)
                     return dest_path
-                except Exception:
+                except Exception as e:
+                    logger.warning(
+                        "Building wheel for %s failed: %s",
+                        req.name, e,
+                    )
                     pass
```

Comment:
Minor suggestion, since the block now has other content

```suggestion
```

Reply:
@chrahunt done